server/api/users.js

```diff
@@ -53,3 +53,68 @@ router.get('/:id/latestguesses', async (req, res, next) => {
 		next(error)
 	}
 })
+
+const getRandomInt = max => {
+	return Math.floor(Math.random() * max)
+}
+
+const completedPrevGameOrStartNewGame = async (req, res, next) => {
+	try {
+		let latestWordle = await WordleGame.findAll({
+			where: { userId: req.params.id },
+			include: AcceptedGuess
+		})
+		latestWordle ? (latestWordle = latestWordle[latestWordle.length - 1]) : null
+		if (!latestWordle || latestWordle.acceptedGuesses.length === 5) {
```

Comment:
You ultimately have 6 guesses to guess the wordle word! Update should look like:
`if (!latestWordle || latestWordle.acceptedGuesses.length === 6) {`